user/build-environment-updates/2019-03-27.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Build Environment Update History
+layout: en
+permalink: /user/build-environment-updates/2019-03-27/
+category: linux_build_env_updates
+date: 2019-03-27
+---
+
+# 2019-03-27
+
+## Changes
+- git uses [wire protocol version 2](https://github.com/git/git/blob/master/Documentation/technical/protocol-v2.txt) to do its network operations, like `git fetch` or `git clone`
+- docker makes use of GCE image cache
+- php version aliases are now recognized
+
+## Version updates
```

Comment:
```suggestion
## Version updates

The following languages and tools preinstalled on Xenial have been updated:
```

shall we mention something with regards to these being the pre-installed tools / languages?